File: rb/test/exists_test.rb
# BluefinShieldconexMgmt SDK exists test

require "minitest/autorun"
require_relative "../BluefinShieldconexMgmt_sdk"

class ExistsTest < Minitest::Test
  def test_create_test_sdk
    testsdk = BluefinShieldconexMgmtSDK.test(nil, nil)
    assert !testsdk.nil?
  end
end

File: swift/Tests/ProjectNameSDKTests/ExistsTest.swift
// BluefinShieldconexMgmt SDK exists test.

import XCTest

@testable import BluefinShieldconexMgmtSdk

final class ExistsTest: XCTestCase {
  func testMode() {
    let testsdk = BluefinShieldconexMgmtSDK.testSDK(nil, nil)
    XCTAssertEqual(testsdk.mode, "test")
  }
}
